utility/src/main/java/org/oppia/android/util/parser/html/PolicyPageTagHandler.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.util.parser.html
+
+import android.text.Editable
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ClickableSpan
+import android.view.View
+import org.oppia.android.app.model.PolicyPage
+import org.oppia.android.util.logging.ConsoleLogger
+import org.xml.sax.Attributes
+
+/** The custom tag corresponding to [PolicyPageTagHandler]. */
+const val CUSTOM_POLICY_PAGE_TAG = "oppia-noninteractive-policy"
+private const val PRIVACY_POLICY_PAGE = "privacy"
+private const val TERMS_OF_SERVICE_PAGE = "tos"
+private const val PRIVACY_POLICY = "Privacy Policy"
+private const val TERMS_OF_SERVICE = "Terms of Service"
```

Comment:
Why are these needed?

Reply:
To create spannable string we need it. Instead of passing strings in the logic I created constants.